ipn/ipnlocal/c2n.go

```diff
@@ -207,6 +216,28 @@ func (b *LocalBackend) handleC2NUpdatePost(w http.ResponseWriter, r *http.Reques
 	}()
 }
 
+func (b *LocalBackend) handleC2NPostureIdentityGet(w http.ResponseWriter, r *http.Request) {
+	b.logf("c2n: GET /posture/identity received")
+
+	res := tailcfg.C2NPostureIdentityResponse{}
+
+	// TODO(kradalby): Use syspolicy + envknob to allow Win registry,
+	// macOS defaults and env to override this setting.
+	if b.Prefs().PostureChecks() {
+		sns, err := posture.GetSerialNumbers()
+		if err != nil {
+			b.logf("c2n: failed to get serial number %s", err)
```

Comment:
Do you want to `return` here, or at least not set `res.SerialNumbers`?

While `C2NPostureIdentityResponse` only contains serial numbers, we could also return a 500 here with the error message.